src/components/sidebar/nodeView/NodeView.tsx

```diff
@@ -198,7 +228,6 @@ class NodeView extends ViewFormBase<INodeViewProps, INodeViewState> {
 
     private onNodeGeometryChange = (property: NodeLocationType, value: any) => {
         this.props.nodeStore!.updateNodeGeometry(property, value, NodeMeasurementType.Measured);
-        this.validateProperty(nodeValidationModel[property], property, value);
     };
 
     private onChangeNodeProperty = (property: keyof INode) => (value: any) => {
```

Comment:
Hmm, maybe this line 206 can be removed and also maybe lines 207-209
Reason: now validation is done through watcher on each property change

Reply:
Removed lines 235-238. Validating seems to be working just fine without them